### PR TITLE
Desktopversion

### PR DESCRIPTION
**Pull request by Ade179**
### first Capstone Project

- Below is a link to my video on the project
https://www.loom.com/share/92e34b4efecf4f419b52b7e604c2b060
- linters have been set up correctly
- correct GitHub workflow has been utilised
- Built with HTML CSS and javascript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            <div class="desktoplighterheader">
+                <div class="desktopheader">AALA 2022</div>
+                <nav class="references">
+                    <a href="about.html" class="about">About</a>
+                    <a href="#mainprogramme" class="programme">Program</a>
+                    <a href="">Join</a>
+                    <a href="#sec4">Sponsor</a>
+                    <a href="">News</a>
+                </nav>
+            </div>
```
Comment (awais-amjed) [inline index.html:37]: Kindly fix the `font-family` here 🤗

Your design:

image

Template design:

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            </nav>
+
+        </article>
+
+
+        <section class="sec3">
+            <div class="guestscontainer">
+
+            </div>
+        </section>
```
Comment (awais-amjed) [inline index.html:133]: Kindly fix the `font-family` for these two sections too 🤗
It should be `Lato`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                    <li class="partnerlist">Tama</li>
+                </ul>
+            </div>
+
+        </section>
+
+        <footer class="indexfooter">
+            <div class="footercontainer">&#169 AALA 2022</div>
+            <div class="footertext">2022 Annual Arts And Literary Appreciation</div>
+        </footer>
```
Comment (awais-amjed) [inline index.html:152]: Kindly fix the font-family for this section also 🤗

Comment (awais-amjed) [inline index.html:152]: Kindly fix the `font-family` for this section also 🤗

File: about.html [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    <section class="sec1" id="sec1">
+          <button class="btn"><i class="fa-solid fa-bars navbtn"></i></button>
+        <div class="desktopdarkheader">
+            <div class="dark-items">
+                <ul>
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><img src="images/twitter.png" alt="twitter-image"></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
```
Comment (awais-amjed) [inline about.html:34]: Kindly fix the `font-family` for these also 🤗

File: about.html [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+        </div>
+        <div class="desktoplighterheader">
+            <div class="desktopheader">AALA 2022</div>
+
+            <nav class="references">
+                <a href="#mainprogramme" class="programme">Program</a>
+                <a href="" class="join">Join</a>
+                <a href="#sec4" class="sponsor">Sponsor</a>
+                <a href="" class="news">News</a>
+                <a href="index.html" class="news">Home</a>
```
Comment (awais-amjed) [inline about.html:46]: Kindly fix the `font-family` for this too 🤗

File: about.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+                        <div id="img1txt">AALA Global Summit in belarus</div>
+                    </div>
+                </div>
+                <div class="aboutimg2">
+                    <div class="textcontz">
+                        <div class="imgtxt2">2015</div>
+                        <div class="imgtxt2">AALA Global Summit in, Nigeria</div>
+                    </div>
+                </div>
+            </div>
```
Comment (awais-amjed) [inline about.html:106]: Kindly fix the `font-family` for the text over the images 🤗

image

File: index.html [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
+
+            <nav class="wholepage">
+                <a href="" id="wholepage">SEE THE WHOLE PAGE</a>
+            </nav>
+
+        </article>
```
Comment (awais-amjed) [inline index.html:126]: Kindly add some spacing around this section as shown in the design 🤗

Your design:

image

Template design:

image

File: index.html [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+                    <h3 class="cont-text">Extras</h3>
+                    <div class="cont-description">There will be prizes to go
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
+
+            <nav class="wholepage">
+                <a href="" id="wholepage">SEE THE WHOLE PAGE</a>
+            </nav>
```
Comment (awais-amjed) [inline index.html:124]: Kindly note that this button is missing on your mobile design 🤗

Your design:

image

Template design:

image

**Review by awais-amjed — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️

Hi @Ade179 , 

Good job so far! 👏👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🥳🙌🤗

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] No Linter issues ✔️ 
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Required Changes ♻️

- [ ] I noticed that you created a Capstone Project last week too that got approved and this is a continuation of that Project. So, you don't need to change much, just a few minor details I have mentioned below to make your project look even better. Kindly apply them and you should be ready for approval 🤗
- [ ] Kindly add the missing links in your Readme file 😃
image
- [ ] Kindly use the same `font-family` provided in the design everywhere to make your whole website look uniform 🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Ade179) [reply index.html:37]: lato added, thanks

File: index.html (hunk 2 above)
Comment (Ade179) [reply index.html:133]: Lato added thanks

File: index.html (hunk 3 above)
Comment (Ade179) [reply index.html:152]: Lato added thank you!!

Comment (Ade179) [reply index.html:152]: done

File: about.html (hunk 4 above)
Comment (Ade179) [reply about.html:34]: done, thanks

File: about.html (hunk 5 above)
Comment (Ade179) [reply about.html:46]: done, thanks

File: about.html (hunk 6 above)
Comment (Ade179) [reply about.html:106]: done, thanks

File: index.html (hunk 8 above)
Comment (Ade179) [reply index.html:124]: done, thanks

**Pull request merged**